content/core-features/occupancy.textile

```diff
@@ -0,0 +1,17 @@
+---
+title: Occupancy
+section: core-features
+index: 29
+meta_description: "The occupancy feature returns metrics relating to clients connected to a channel."
```

Comment:
"connected to a channel": do we use this terminology generally? I think more accurately we would say "attached to a channel"

Reply:
No, that's a good point. I've updated this to "attached" in https://github.com/ably/docs/pull/1232